### PR TITLE
chore: update to rules_js 1.0.0-rc.4

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: angular/defs.bzl [hunk 1]
```diff
@@ -134,7 +134,10 @@ def ng_app(name, project_name = None, deps = [], test_deps = [], **kwargs):
         name = "test",
         chdir = native.package_name(),
         args = ["%s:test" % project_name],
+        include_declarations = True,
```
Comment (jbedard) [inline angular/defs.bzl:137]: We need declarations for tests? 🤔

File: angular/defs.bzl [hunk 2]
```diff
@@ -134,7 +134,10 @@ def ng_app(name, project_name = None, deps = [], test_deps = [], **kwargs):
...
         data = srcs + test_srcs + deps + test_deps + TEST_DEPS + TEST_CONFIG + COMMON_CONFIG,
+        # TODO: fix in sandbox
```
Comment (jbedard) [inline angular/defs.bzl:139]: What fix is required? Is there a link we can put here?

File: angular/defs.bzl (hunk 1 above)
Comment (gregmagolan) [reply angular/defs.bzl:137]: Whoops. Shouldn't here. Snuck in from debugging the test failure.

File: angular/defs.bzl (hunk 2 above)
Comment (gregmagolan) [reply angular/defs.bzl:139]: I don't know yet. Needs more investigation.

**Review by jbedard — APPROVED** [review]

File: angular/defs.bzl (hunk 2 above)
Comment (gregmagolan) [reply angular/defs.bzl:138]: Couldn't figure it out quickly. Will need to do some regression testing on this one to find out what commit broke it.

**Pull request merged**